pkg/scannerv4/client/client.go

```diff
@@ -191,6 +195,10 @@ func (c *gRPCScanner) IndexAndScanImage(ctx context.Context, ref name.Digest, au
 	return vr, nil
 }
 
+func (c *gRPCScanner) GetMatcherMetadata(ctx context.Context) (*v4.Metadata, error) {
+	return c.matcher.GetMetadata(ctx, &types.Empty{})
```

Comment:
It may be worth adding a retry mechanism like the other calls so we can be more resilient. If done, we may want to update the context's timeout to be longer

Reply:
Added.  I didn't believe it warranted a retry because it's a 'localish' call with no external dependencies (we added retries to the other calls to account for registry issues), the UI (the only consumer I'm aware of) polls this endpoint continuously as it is, and we didn't have retries for the same for Scanner V2.